manager/state/raft/raft.go

```diff
@@ -992,10 +994,15 @@ func (n *Node) updateNodeBlocking(ctx context.Context, id uint64, addr string) e
 	if m == nil {
 		return errors.Errorf("member %x is not found for update", id)
 	}
+	if m.BackupAddrs == nil {
+		m.BackupAddrs = make(map[string]*types.Timestamp)
+	}
+	m.BackupAddrs[m.Addr] = ptypes.MustTimestampProto(time.Now())
```

Comment:
Here idea is that there is no `m.Addr` in `BackupAddrs`. Overall using map looks clumsy now :/